packages/cssnano-utils/src/__tests__/sameParent.test.js

```diff
@@ -1,8 +1,15 @@
 import postcss from 'postcss';
 import sameParent from '../sameParent';
 
+const plugin = () => {
```

Comment:
Isn't there another way to suppress the warnings? In fact we want postcss to do nothing here. I think adding more code to the test does not help with clarity.

Reply:
Well... my rationale was that when I opened up that file, not knowing the code at all, it was not clear to me at first that you did in fact not want postcss to do anything. So although the dummy plugin at the top of the file does add a little bit of code - could it not also help point out the intention by explicitly passing in a "dummy" instead of nothing?

I didn't see any obvious alternative way (e.g. an option for postcss), but admittedly I also didn't look super hard. Would be happy to have another look though

Reply:
Yeah, I think just adding a comment to explain what's going on is the best solution.

Reply:
Alright - I'll look into it a bit more thoroughly and see if I find a better solution to suppress the warnings then :-)

Reply:
Ah, so there is actually an option to suppress it:

https://github.com/postcss/postcss/blob/10d974f81d2b925a9e8c0912857dcda1bf6720c0/lib/processor.js#L19

So;

```
test('should calculate same parent', () => {
  return postcss()
    .process('h1 {} h2 {}', { from: undefined, hideNothingWarning: true })
    .then((result) => {
      (...)
    });
});
```

Works!